src/Angular.js

```diff
@@ -675,6 +676,28 @@ var isArray = Array.isArray;
 
 /**
  * @ngdoc function
+ * @name angular.isError
+ * @module ng
+ * @kind function
+ *
+ * @description
+ * Determines if a reference is an `Error`.
+ *
+ * @param {*} value Reference to check.
+ * @returns {boolean} True if `value` is an `Error`.
+ */
+function isError(value) {
+  var tag = ({}).toString.call(value);
```

Comment:
We already have a closure-global `toString = Object.prototype.toString`. You can simply `toString.call(value)`.